### DISCUSSION TITLE
Postgres inbuilt functions

### DISCUSSION THREAD

**Discussion by kesavamuthu** [prisma/prisma#6113] [Q&A]
how to use PostgreSQL inbuilt functions like regexp_replace() in prisma ?

I'm using prisma v2.15.0

**Comment by ryands17** [comment]
@kesavamuthu 👋 

Currently you would only be able to use this with Prisma's [raw query](https://www.prisma.io/docs/concepts/components/prisma-client/raw-database-access) methods.

I would suggest adding a 👍 on this request: https://github.com/prisma/prisma/issues/5761 so that we can look at the priority for this :)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
